odoo/models.py

```diff
@@ -5738,6 +5748,16 @@ def __iter__(self):
             for id in self._ids:
                 yield self._browse(self.env, (id,), self._prefetch_ids)
 
+    def __reversed__(self):
+        """ Return an reversed iterator over ``self``. """
+        if len(self._ids) > PREFETCH_MAX and self._prefetch_ids is self._ids:
+            for ids in self.env.cr.split_for_in_conditions(reversed(self._ids)):
+                for id_ in ids:
+                    yield self._browse(self.env, (id_,), ids)
+        else:
+            for id_ in reversed(self._ids):
+                yield self._browse(self.env, (id_,), reversed(self._prefetch_ids))
```

Comment:
Seems broken as `reversed` is an iterator, it's not reversible, so the first time it's used it's going to be consumed, and if it's used as second time it'll be empty.

Reply:
Yes, my bad. I redo fixed version